Remote work & communication guidelines

krantzinator opened the pull request:
- addresses #33 and has information that could be shared with #53 
- @blacknight811 & I will be incorporating these guidelines into our workflow over the next week, and will report back with any recommended changes

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, the dev team is ~20% full-time remote. Given that, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, ...
```
Specify that the 20% number is FTEs. Also mention that the majority of our contractors are remote and so it is important to have solid remote protocols.

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+- ensure company meetings have good audio for online attendees
+- regular 1-on-1 communication with same-department coworkers for pairing on non-client coding projects
+- fly remote employees to the Phoenix office roughly once/quarter for 3-5 days, covering the flight, hotel, and meals (and try to schedule those trips at the same time for the remotees!)
+- send gift cards to remote employees when they miss out on a project team's celebration
```
Cranky grammarian comment: For consistency, we've been using asterisks for unordered lists, and capitalizing the first word on a bullet point, and also ending them with periods. 😁

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+
+Things we'd like to do:
+- integrate better screensharing and pair-programming tools
+- make the inclusion of a GTM or Google Meet the default when creating meetings
```
We're moving away from GTM to Zoom... Maybe rephrase as the generic "video conference software"?

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+Things we'd like to do:
+- integrate better screensharing and pair-programming tools
+- make the inclusion of a GTM or Google Meet the default when creating meetings
+- have a shared microphone in larger company meetings that can be passed around, or have a culture where questions that are asked from the room are repeated into the central mic for the benefit of remote attendees
```
I dunno about this one... The majority of our meetings don't have a ton of people in one room. It's only for all-hands that this is an issue (I think?). The A/V issues also get complex... I'd file this as a nice-to-have but realistically wouldn't make a move on it for a while longer.

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+
+## General Communication
+
+- Set your notification preferences
```
This goes beyond remote, imo. Everyone in office or not needs to be mindful about their notifications... Consider moving it to its own page altogether.

krantzinator replied:
@swilliams it's own page or to the Operations page?

swilliams commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,58 @@
+As an organization, we are working to make remote more integrated.
+
+Things we currently do are:
+* ensure company meetings have good audio for online attendees
```
Capitalize bullet sentences and end with periods. 😃

krantzinator replied:
@swilliams does that include the "header" bullets, like `* Set your notification preferences`?

swilliams replied:
If they're headers they should be header tags (and then title-cased). Like `## Set Your Notification Preferences`.

krantzinator commented:
@TinaHeiligers @swilliams I moved a couple of things to a `Communication` page, and added a new paragraph there as well titled `### Alert your team if you usually work outside of 9-to-5 hours`.

swilliams commented:
Just realized that the `Communication` page is missing. @krantzinator can you add it lickety-split? 😄

krantzinator commented:
@swilliams doh I pushed it to the wrong branch. PR incoming.

vinnieking06 commented on docs/Team/Working-Remote.md:
```diff
@@ -0,0 +1,52 @@
+* Make the inclusion of video conferencing the default when creating meetings.
+
+Nice-to-haves as we grow:
+* For all-hands meetings or company standups, have a shared microphone that can be passed around, or have a culture where questions that are asked from the room are repeated into the central mic for the benefit of remote attendees.
```
I like this.
